seed/models/column_mapping_profiles.py

```diff
@@ -19,7 +18,7 @@ class ColumnMappingProfile(models.Model):
     )
 
     name = models.CharField(max_length=255, blank=False)
-    mappings = JSONField(default=list, blank=True)
+    mappings = models.JSONField(default=dict, blank=True)
```

Comment:
the default should have been a `dict`, tested this change and didn't see any issues with it.